tests/bluetooth/tester/src/audio/btp_bap.c

```diff
@@ -349,11 +349,6 @@ static const struct btp_handler bap_handlers[] = {
 		.expect_len = sizeof(struct btp_bap_discover_cmd),
 		.func = btp_bap_discover,
 	},
-	{
-		.opcode = BTP_BAP_SEND,
-		.expect_len = BTP_HANDLER_LENGTH_VARIABLE,
-		.func = btp_bap_audio_stream_send,
-	},
```

Comment:
Was not sure whether to remove this or not, since we are still receiving the commands from the autopts client, and now we will always return an error. "Luckily" it does not check the response code, so this doesn't really affect anything in autopts

Reply:
btp errors are handled via exceptions in autopts, so this will affect wid handler

I'd leave this as empty sub that return success for now (at least until https://github.com/auto-pts/auto-pts/issues/1251 is properly implemented in autopts)

Reply:
They are passing regardless, but I'll leave the handler empty for now